src/shogun/kernel/ExponentialARDKernel.h

```diff
@@ -27,55 +27,57 @@ enum EARDKernelType
 	KT_FULL=30
 };
 
-/** @brief Linear Kernel with Automatic Relevance Detection computed
+/** @brief Exponential Kernel with Automatic Relevance Detection computed
  * on CDotFeatures.
  *
  * It is computed as
  *
  * \f[
- * k({\bf x},{\bf y})= ||{\bf x}-{\bf y}||
+ * k({\bf x},{\bf y})= exp(-||{\bf x}-{\bf y}||)
```

Comment:
pls use "\exp" when in latex math mode
pls use "\Vert" rather than "||"

(also grep for it in the docs and change all others if possible)